script/bootstrap

```diff
@@ -9,4 +9,6 @@ cd "$(dirname "$0")/.."
 echo "Installing development dependencies..."
 uv pip install wheel --constraint homeassistant/package_constraints.txt --upgrade
 uv pip install colorlog $(grep awesomeversion requirements.txt) --constraint homeassistant/package_constraints.txt --upgrade
-uv pip install -r requirements_test.txt -c homeassistant/package_constraints.txt --upgrade
+uv pip install -r requirements_test_all.txt -c homeassistant/package_constraints.txt --upgrade
+uv pip install -r requirements_test_pre_commit.txt -c homeassistant/package_constraints.txt --upgrade
+uv pip install -r requirements_all.txt -c homeassistant/package_constraints.txt --upgrade
```

Comment:
This one makes `requirements_test_all` uneeded, and should be added for the same reasons.